test/tests/markers/reporter_point_marker/point_marker_test.i

```diff
@@ -0,0 +1,62 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  nz = 0
+  zmax = 0
+  elem_type = QUAD4
+[]
+
+[Problem]
+  solve = false
+[]
+
+[Executioner]
+  type = Steady
+[]
+
+[Reporters]
+  [coords]
+    type=ConstantReporter
+    real_vector_names = 'x y z'
+    real_vector_values = '.31 .41 .51  .31 .41 .51 .31 .41 .51 .8;
+                          .31 .31 .31 .41  .41 .41 .51 .51 .51 .8;
+                           0   0   0  0   0   0  0  0   0  1;'
+    outputs=none
+  []
+  [bad_coords]
+    type=ConstantReporter
+    real_vector_names = 'x y z'
+    real_vector_values = '.31 .41 .51;
+                          .31 .31 .31 .41  .41 .41 .51 .51;
+                           0   0   0  0   0   0  0  0   0  1;'
+    outputs=none
+  []
+[]
+
+[Adaptivity]
+  [Markers]
+      inactive = 'bad_coord'
```

Comment:
You can do this, but I found it confusing when looking at the tests file. A better alternative would be the `active` parameter